pom.xml

```diff
@@ -13,6 +13,7 @@
 		<groupId>org.springframework.data.build</groupId>
 		<artifactId>spring-data-parent</artifactId>
 		<version>2.1.0.BUILD-SNAPSHOT</version>
+		<relativePath>../spring-data-build/parent/pom.xml</relativePath>
```

Comment:
What's the reason we add this? We usually don't use relative paths to make sure the lookup of the parent is always done from the repository. Also, the declaration assumes that the parent project has been checked out into a folder of that exact name, which might not be correct. Also, if that folder has the project checked out in a different branch, it might see a different pom.